### PR TITLE
Soap can now be emagged.

### PR DESCRIPTION
**Pull request by brndd**
What it says on the tin. It gains no special properties other than a syndicate appearance.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
It's a fucking electromagnetic pulse

**Comment by brndd** [comment]
all chemistry is fundamentally electromagnetism

File: code/defines/obj/weapon.dm [hunk 1]
```diff
@@ -85,6 +85,26 @@
 	throwforce = 0
 	throw_speed = 4
 	throw_range = 20
+	var/probed = 0
+
+/obj/item/weapon/soap/attackby(obj/item/weapon/W, mob/user)
```
Comment (ShiftyRail) [inline code/defines/obj/weapon.dm:90]: I'm pretty sure there is an `emagged` proc for that kind of interactions

**Comment by kilozombie** [comment]
Isn't there literally syndie soap you can just buy?
Dumbest PR of the year

**Comment by Irradiation** [comment]
You can emag wooden marracas to make them explosive. You can't make this shit up.

**Comment by brndd** [comment]
@kilozombie 
>Isn't there literally syndie soap you can just buy?

Yes. Emagging soap allows you to turn regular soap into the same syndie soap you can order through the uplink. No self-respecting agent would wash themselves with filthy NanoTrasen soap, but sometimes supplies in the field are limited. This PR solves that.

**Comment by kilozombie** [comment]
> Yes. Emagging soap allows you to turn regular soap into the same syndie soap you can order through the uplink. No self-respecting agent would wash themselves with filthy NanoTrasen soap, but sometimes supplies in the field are limited. This PR solves that.

Why did you do this

What did you think this would add to the game, it's absolutely pointless and basically dummies out something you can buy in the uplink

**Comment by brndd** [comment]
>What did you think this would add to the game

Whimsy. It's not a serious game. This is also why I didn't include a changelog, because it'll be more fun to discover something like this by chance.

>it's absolutely pointless and basically dummies out something you can buy in the uplink

The soap exists in the uplink as a meme item, much like the syndicate balloon. This is just another meme way to acquire the same meme item. Which, I should remind everyone, is completely identical to regular soap in functionality, and can also be found in several locations on many (all?) of the stations at roundstart.

**Comment by kilozombie** [comment]
This is like if you could emag any balloon to make it a syndie balloon, it's just dumb for no reason and it isn't "whimsy". Emags can already do a ton of stuff, please DNM.

**Comment by brndd** [comment]
That would be a valid concern if the syndicate soap held the same kind of iconic status as a troll item as the balloon (which consumes all your telecrystals), but it does not.

**Comment by Accipitermaris** [comment]
What if instead using regular soap on an emag turned it into a regular ID card with no name bank account or access

You’ve “cleaned it up”!

**Comment by ShiftyRail** [comment]
@Accipitermaris I know space operates on crazy logic but come on

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
>You can emag wooden marracas to make them explosive. You can't make this shit up.

This was a huge mistake and never should have been allowed.

**Comment by Exxion** [comment]
In 2562, there are no more beans, so the maracas are electronic

**Comment by MadmanMartian** [comment]
No

**Pull request closed without merging**